frontend/public/locales/zh/cluster-settings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Invalid cluster version": "无效的集群版本",
+  "Cancel update": "取消更新",
+  "Available updates": "可用更新",
+  "Failing": "失败",
+  "View conditions": "查看条件",
+  "Update to {{version}} in progress": "更新到{{version}}进行中",
+  "No update channel selected": "未选择更新频道",
+  "Version not found": "找不到版本",
+  "Error retrieving": "检索错误",
+  "Up to date": "最新",
+  "View release notes": "查看发行注记",
+  "Unknown": "未知",
+  "None": "无",
+  "Current version": "当前版本",
+  "Last completed version": "最后完成的版本",
+  "Learn more about OpenShift update channels": "了解有关 OpenShift 更新频道的更多信息",
+  "Channel": "频道",
+  "Channels help to control the pace of updates and recommend the appropriate release versions. Update channels are tied to a minor version of OpenShift Container Platform, for example 4.5.": "使用频道有助于控制更新的速度并推荐适当的发行版本。更新通道与 OpenShift Container Platform的次版本（例如 4.5）相关联。",
+  "Version": "Version",
```

Comment:
"Version" need be translated.

Reply:
This has not been updated.

Reply:
Checked the file, this was actually addressed in the translation file - will send the file again just in case.

Reply:
Version is now updated.